packages/wrangler/src/bundle.ts

```diff
@@ -5,6 +5,8 @@ import * as esbuild from "esbuild";
 import makeModuleCollector from "./module-collection";
 import type { CfModule, CfScriptFormat } from "./api/worker";
 
+export type Entry = { file: string; directory: string };
```

Comment:
Can you add some jsdoc comments here. For example it is not clear to me whether the `file` is relative to the `directory` or not.